listenbrainz/webserver/views/test/test_user.py

```diff
@@ -12,7 +18,23 @@ def setUp(self):
         self.user = db_user.get_or_create('iliekcomputers')
         self.weirduser = db_user.get_or_create('weird\\user name')
 
+        self.log = logging.getLogger(__name__)
+        influx = InfluxDBClient(host=config.INFLUX_HOST, port=config.INFLUX_PORT, database=config.INFLUX_DB_NAME)
+        influx.query('''create database %s''' % config.INFLUX_DB_NAME)
+
+        self.logstore = init_influx_connection(self.log, {
+            'REDIS_HOST': config.REDIS_HOST,
+            'REDIS_PORT': config.REDIS_PORT,
+            'INFLUX_HOST': config.INFLUX_HOST,
+            'INFLUX_PORT': config.INFLUX_PORT,
+            'INFLUX_DB_NAME': config.INFLUX_DB_NAME,
+        })
+
     def tearDown(self):
+        influx = InfluxDBClient(host=config.INFLUX_HOST, port=config.INFLUX_PORT, database=config.INFLUX_DB_NAME)
```

Comment:
Creating individual clients for both setUp and tearDown would be a bit inefficient, I guess.